pyomo/contrib/mindtpy/plugins.py

```diff
@@ -1,2 +1,3 @@
+# -*- coding: utf-8 -*-
```

Comment:
I don't know if we need it on EVERY file, but ok. You followed the instructions to the letter!